### PR TITLE
[ci skip] chore: small fixes to comments and features

### PR DESCRIPTION
**Pull request by kristofbolyai**
Some fixes that are not worth pushing a new build or creating a new PR..

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/ui/CustomTradeMarketResultScreenFeature.java [hunk 1]
```diff
@@ -23,6 +25,7 @@
 import net.minecraft.world.item.ItemStack;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
+@ConfigCategory(Category.UI)
```
Comment (ShadowCat117) [inline common/src/main/java/com/wynntils/features/ui/CustomTradeMarketResultScreenFeature.java:28]: Not trade market?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/ui/CustomTradeMarketResultScreenFeature.java:28]: Well all custom screens are here, as well as the feature itself is in this package

Comment (ShadowCat117) [reply common/src/main/java/com/wynntils/features/ui/CustomTradeMarketResultScreenFeature.java:28]: True, just most people I've seen searching to disable it have expected it to be in trade market category

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/ui/CustomTradeMarketResultScreenFeature.java:28]: Kind of have to rework categories either way..

**Pull request merged**